packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st

```diff
@@ -7,13 +7,16 @@ keyboardShortcuts
 		contents: 
 'Structuring
 Ctrl-Shift-[1,2,...,0]	toggle 1st, 2nd, ... or 10th structure from the top of the structure list
+(NOTE: This works only when using an american keyboard layout)
```

Comment:
Should propably fix that by creating another preference and different shortcut string

Reply:
good point

Reply:
See #178